src/app/simple-usage/simple-usage.component.spec.ts

```diff
@@ -92,7 +92,7 @@ describe( 'SimpleUsageComponent', () => {
 		} );
 
 		it( 'should be synced with editorData property', () => {
-			getEditorNamespace( ckeditorComponents[ 0 ].editorUrl )
+			return getEditorNamespace( ckeditorComponents[ 0 ].editorUrl )
```

Comment:
It's kind of bad moment for introducing test changes during dependencies updates, as it may suggest something started failing, but I see that this test has been written incorrectly. I suppose you just spot the issue and refactored an invalid test?

Reply:
Yup - I suppose the newer Jasmine version is less tolerant for the bad tests. Last time I was updating deps there was also a number of tests that started to fail because they were simply wrong and now this one test needs a little amendment. I didn't think that it's worth creating an issue but if you think it would be a better place to fix it we can do it this way.